integrations-and-supported-tools/tensorboard/notebooks/Neptune_Tensorflow_Tensorboard.ipynb

```diff
@@ -0,0 +1,488 @@
+   "metadata": {
+    "id": "FUaDrmRu-h3k"
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "If you want to see the example logged to your own workspace instead:\n",
+    "\n",
+    "  1. Create a Neptune account. [Register &rarr;](https://neptune.ai/register)\n",
+    "  1. Create a Neptune project that you will use for tracking metadata. For instructions, see [Creating a project](https://docs.neptune.ai/setup/creating_project) in the Neptune docs."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w_0KXqlePqNi"
+   },
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i36OvZAgvxkj"
+   },
+   "outputs": [],
+   "source": [
+    "%pip install -U neptune neptune-tensorboard==1.0.0rc4 tensorflow --no-cache-dir"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "rik2HDr0r5Hn",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "## Start a run\n",
+    "\n",
+    "To create a new run for tracking the metadata, you tell Neptune who you are (`api_token`) and where to send the data (`project`).\n",
+    "\n",
+    "You can use the default code cell below to create an anonymous run in a public project. **Note**: Public projects are cleaned regularly, so anonymous runs are only stored temporarily.\n",
+    "\n",
+    "### Log to your own project instead\n",
+    "\n",
+    "Replace the code below with the following:\n",
+    "\n",
+    "```python\n",
+    "import neptune\n",
+    "from getpass import getpass\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    project=\"workspace-name/project-name\",  # replace with your own (see instructions below)\n",
+    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "To find your API token and full project name:\n",
+    "\n",
+    "1. [Log in to Neptune](https://app.neptune.ai/).\n",
+    "1. In the bottom-left corner, expand your user menu and select **Get your API token**.\n",
+    "1. The workspace name is displayed in the top-left corner of the app. To copy the project path, in the top-right corner, open the settings menu and select **Properties**.\n",
+    "\n",
+    "For more help, see [Setting Neptune credentials](https://docs.neptune.ai/setup/setting_credentials) in the Neptune docs.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "XMdpj-Se4t0U",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    api_token=neptune.ANONYMOUS_API_TOKEN,  # replace with your own\n",
+    "    project=\"common/quickstarts\",  # replace with your own\n",
```

Comment:
Isn't there a dedicated integration project?

Reply:
there is one now: common/tensorboard-integration